lib/vanagon/component.rb

```diff
@@ -231,16 +233,25 @@ def get_build_dir
     # Fetches secondary sources for the component. These are just dumped into the workdir currently.
     #
     # @param workdir [String] working directory to put the source into
-    def get_sources(workdir)
+    def get_sources(workdir) # rubocop:disable Metrics/AbcSize
```

Comment:
We need this comment to tell rubocop that's it's okay this method is failing checks and to not error out because of it